tracer/test/Datadog.Trace.TestHelpers.SharedSource/VerifyHelper.cs

```diff
@@ -220,21 +222,21 @@ public static void AddSimpleScrubber(this VerifySettings settings, string oldVal
             ];
 
             return tags
-                  .Where(kvp => !ignoreKeys.Contains(kvp.Key))
+                 ?.Where(kvp => !ignoreKeys.Contains(kvp.Key))
```

Comment:
This is the main fix. `ScrubNumericTags()` was added in #5562 and the `tags` argument is `null` in some tests. I made the same change in other methods to be safe.